examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -1034,8 +1083,12 @@ def main(args):
         vae_path,
         subfolder="vae" if args.pretrained_vae_model_name_or_path is None else None,
         revision=args.revision,
-        variant=args.variant,
+        variant=args.variant if "playgroundai" not in args.pretrained_model_name_or_path else None,
```

Comment:
This looks a bit strange to me. An alternative would be to just recommend users to use the main variant for this model (they can still use mixed-precision training).

Reply:
+1

Reply:
Done in c8ed8af8. LMK.